themes/bootstrap3/templates/header.phtml

```diff
@@ -18,7 +18,8 @@
           <?php $cart = $this->cart(); if ($cart->isActive()): ?>
             <li id="cartSummary">
               <a id="cartItems" data-lightbox title="<?=$this->transEscAttr('View Book Bag')?>" href="<?=$this->url('cart-home')?>">
-                <i class="fa fa-suitcase" aria-hidden="true"></i> <strong><?=count($cart->getItems())?></strong> <?=$this->transEsc('items')?>
+                <?=$this->icon('cart') ?>
```

Comment:
Is there a reason this change is here, rather than in #1962?

Reply:
These were for tests and were accidentally committed. Removing now.